scim/Dependencies.toml

```diff
@@ -5,11 +5,12 @@
 
 [ballerina]
 dependencies-toml-version = "2"
+distribution-version = "2201.5.0"
```

Comment:
There is a version mismatch with the Ballerina.toml https://github.com/ballerina-platform/module-ballerinax-scim/blob/ada2638465f55c393b1f3aca2db0b8fce256684f/scim/Ballerina.toml#L5 and Dependency.toml. Which Ballerina.toml has a 2201.4.1 version and Dependecy.toml has a 2201.5.0. Shall we update the toml  with the Ballerina version that is going to use for generating and publishing the connector. ?

@LakshanSS  appreciate if you can provide some insight regarding the `Dependncy.toml` version update.

Reply:
Added a comment https://github.com/ballerina-platform/module-ballerinax-scim/pull/4#discussion_r1176188602